build.gradle.kts

```diff
@@ -12,3 +12,21 @@ buildscript {
         classpath("de.mannodermaus.gradle.plugins:android-junit5:1.10.0.0")
     }
 }
+subprojects {
```

Comment:
Does this have to be in the top level Gradle file? Since we are building a single module application ATM it probably doesn't really matter, but in that case I'd put everything into the module gradle file.

Reply:
Oh okay, maybe we can move it then. I just followed the instructions [here](https://developer.android.com/develop/ui/compose/performance/stability/diagnose#setup)